### ISSUE TITLE
App - navn/innhold

### ISSUE THREAD

**Issue by leloberg** [RosenborgSupporterSoftware/RosenborgSupporterSoftware.github.io#4]
Når man først vil utvikle en app, så er det vel ingen grunn til at den kun skal være dedikert til STAut, mtp. alle andre ideéne som er kommet frem. Dermed kommer spørsmål om navngivning på appen. 
Hadde jo vært fint (og allerede etablert navn i miljøet) og kalle den rbkweb, men det krever både tillatelse fra Thomas, samt at det man presenterer i appen fort blir veldig offisielt levert av rbkweb (og det ønsker man kanskje ikke at f.eks STAut skal være). 

Hva tenker dere? Andre forslag til et generisk fint, kort og fengende app-navn som vil omfavne hva enn vi putter i den i fremtiden?

**Comment by havremunken** [comment]
Enig i at STAut bare blir en i mengden i en sånn sammenheng. Thomas har jo som du er inne på vært litt skeptisk til å ha STAut-info andre steder enn forumet da det fort kan oppfattes som en offisiell del av RBKweb, noe som hadde vært ugunstig hvis BS slo seg vrang. Hvis han kunne tenke seg deg, synes jeg at det passer bra, da RBKweb fort kan bli oppfattet som DEN digitale samleplassen for alt av RBK-info utenom klubben selv.

Forsøk på brainstorming, ingen av disse er forslag, bare frø for å trigge videre tanker:

RBKFan
RBKFanatic
Everything RBK
RBK-oraklet
<3RBK<3
Troillapp
All things Rosenborg

Ugh, jeg er skikkelig fyllesyk, jeg må gå og legge meg nedpå.

**Comment by vemundo** [comment]
Ser på det vi diskuterer her som en felles platform for å utvikle hva det skulle være av RBK supporter relatert software. Vi trenger en felles platform som kan gjøre dette tilgjengelig på mobil men også web.

Så navnet bør reflektere at dette er noe samlende og felles, ett "hjem". Jeg hadde faktisk også Troillapp i hodet før jeg leste det her. Tenker også på navn som kan knyttes til Lerkendal eller Brakka eller andre klubbsymboler. Kanskje platformen/appen kan hete rett og slett "Brakka" ? Ie. fange klubbsjela, et samlingspunkt for RBKere.

**Comment by leloberg** [comment]
Troillapp kan kanskje fungere. Jeg skulle aller helst hatt et navn som henviser til "rbk" eller "rosenborg" på et vis. 

Apper som eksisterer for tilfeldige andre fotballlag:
- FC Barcelona Official App
- Man United App
- Real Madrid App
- Arsenal

Uansett. Dette er jo offisielle apper fra klubben, så det skiller jo seg fra vårt konsept. Allikevel skulle jeg gjerne sett at vi har et navn som blir tatt seriøst, og som er lett å bruke i daglig prat. 
"Hei, har du testet rbkappen" 
"Hei, har du testet Troillapp"

Litt irriterende at vi mest sannsynlig ikke kan benytte rbkweb, som ville vært perfekt navn. :)

PS: Forøvrig er det vel ingen grunn til å ikke involvere de som faktisk vil bruke appen; altså spørre på forumet.

**Comment by vemundo** [comment]
God ide, vi trenger egentlig bare en arbeidstittel intill videre, så kan vi kjøre en tråd/diskusjon/poll på RBKWeb og la folk foreslå navn når vi har kommet så langt at det nærmer seg noe å vise frem.

**Comment by leloberg** [comment]
Ja, egentlig ikke noe stress. Jeg har gjerne brukt appnavnet som prosjektnavn tidligere også, men det trenger jeg jo egentlig ikke. Kan bruke rosenborgsuppsoft eller noe sånt i manifest og core code.
